src/gcode-machine-control_test.cc

```diff
@@ -35,6 +35,9 @@ static void init_test_config(struct MachineControlConfig *c,
 }
 
 namespace {
+// Record Segment queue calls we care avout.
+enum { CALL_wait_queue_empty, NUM_COUNTED_CALLS };
```

Comment:
I am not entirely sure if it needs so complicated with enum, a Count() function and a publically accessible array.
Couldn't that just be a simple int ?

(it looks like this is prepared for 'future use'. But we should decide that when the future is there and we need 10 different counters).

Reply:
(done)